meta-resin-common/recipes-support/resin-unique-key/resin-unique-key/resin-unique-key

```diff
@@ -43,15 +43,16 @@ fi
 
 if [ ! -s "$CONFIG_PATH" ]; then
     echo "[INFO] resin-unique-key: Config file is missing. Creating.."
-    echo '{}' > $CONFIG_PATH
+    echo '{}' > "$CONFIG_PATH"
 else
-    read key <<<$(jq -r ".$key_name //empty" $CONFIG_PATH)
+    read key <<<$(jq -r ".$key_name //empty" "$CONFIG_PATH")
 fi
 
 if [ -z "$key" ]; then
     echo "[INFO] resin-unique-key: $key_name missing from config file. Generating..."
     key=$(openssl rand -hex 16)
-    echo $(cat $CONFIG_PATH | jq ".$key_name=\"$key\"") > $CONFIG_PATH
+    generated_json=$(jq ".$key_name=\"$key\"" "$CONFIG_PATH")
+    echo "$generated_json" > "$CONFIG_PATH"
```

Comment:
This is exactly what I tried to avoid because of https://github.com/koalaman/shellcheck/wiki/SC2094 .